test/TopupEUReTest.t.sol

```diff
@@ -43,26 +45,28 @@ contract TopupTest is BaseFixture {
         delayModule.executeNextTx(EURE, 0, payload, Enum.Operation.Call);
 
         (bool canExec, bytes memory execPayload) = roboModule.checker();
+        (bytes memory dataWithoutSelector, bytes4 selector) = _extractEncodeDataWithoutSelector(execPayload);
 
         assertTrue(canExec);
```

Comment:
shouldnt we also check for `assertFalse(canExec)` at the beginning of the test, to make the assert on L50 meaningful?

Reply:
what i mean here is that L50 in `testExitPool` checks for `canExec` being true, but it doesnt check the state of that var at the start of the function. in other words, maybe `canExec` was true all along. 

if the following were to be included in the beginning of the test, we would actually _prove_ that the transfer out was the thing that was responsible for the state change of `canExec`:
```
(bool initialCanExec,) = roboModule.checker();
assertFalse(initialCanExec);
```

Reply:
included in 5393e3edd431477b0c28b52d06a8ba081d2837a7